### PR TITLE
Add Desmos

### PR DESCRIPTION
**Pull request by sn00py1310**

### CODE REVIEW

File: img/d/desmos.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...498 -64.21062,-57.01762c-20.87722,-13.50879 -41.05269,-20.00003 -63.50886,-20.00003c-17.89476,0 -26.49126,2.10527 -47.71936,12.28072c-18.59652,8.77194 -34.38601,20.35091 -58.2457,42.80708l-14.56142,13.68423l0,-118.24578c0,-128.07035 0,-128.42123 9.4737,-140.70195c2.45614,-3.33334 8.5965,-8.07019 13.68423,-10.52633l9.12282,-4.56141l367.71981,0l367.71981,0l9.12282,4.56141c5.08773,2.45614 11.22809,7.19299 13.68423,10.52633c9.64914,12.45616 9.4737,10.70177 9.4737,163.85988l0,141.75458l-10.87721,7.01755c-14.38598,9.12282 -28.77197,13.33335 -40.00006,11.40352zm-447.89536,-59.29833c22.10529,-28.59653 49.47375,-51.0527 69.47378,-57.3685c19.29827,-5.96492 37.0176,-3.33334 56.31587,8.24563c6.3158,3.85965 11.05265,5.61404 12.28072,4.73685c1.05263,-0.87719 7.01755,-5.78948 13.33335,-10.70177c6.3158,-5.08773 16.66669,-12.45616 23.33337,-16.31581l11.7544,-7.01755l-11.7544,-9.4737c-25.08775,-19.82459 -55.61411,-30.87724 -84.56152,-30.87724c-27.71934,0 -50.17551,7.19299 -75.78958,24.21056c-14.38598,9.4737 -43.68427,37.71935 -59.12289,57.01762l-10.70177,13.33335l20.52634,20.52634c11.22809,11.05265 21.05266,20.17547 21.57898,19.82459c0.70176,-0.17544 6.66668,-7.36843 13.33335,-16.14037z"/></g></svg>
```
Comment (phallobst) [inline img/d/desmos.com.svg]: I think you got that logo from `desmos.com.au`...

Comment (sn00py1310) [reply img/d/desmos.com.svg]: I took the page icon from `desmos.com`, but it was only a png, so I converted it and colored it with the green from the image (without the gradient).

Comment (Carlgo11) [reply img/d/desmos.com.svg]: @sn00py1310 converting PNGs to SVGs rarely works well. I usually google the company name and append `filetype:svg` in the search query.

Here's what I found when I did that for Desmos:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="#136936" d="m127 293 27-26c-20-43-37-88-52-133S74 43 61-3h377l-5 17-7 24a1005 1005 0 0 1-37 121l-16 43c-12 29-24 59-40 87l-14 23c7 10 22 18 33 16l11-18c21-37 37-76 52-116A1664 1664 0 0 0 475 2l1-5h39v264c-2 1-4-1-5-1-24-10-47-6-68 8-13 9-23 20-35 31l-16 15-21 18a36 36 0 0 1-42 2c-12-7-21-17-30-26l-28-28a96 96 0 0 0-39-23c-21-6-40-1-58 10-12 7-22 19-33 29l-35 32c-7 6-15 11-25 11-7 1-14-1-21-4a802 802 0 0 1-55-50l-7-7V-3h26c24 95 55 187 93 277 5 6 6 13 11 19ZM515 305v209H-3V330l12 12c12 11 24 21 39 28 23 11 45 8 65-4 15-8 27-20 39-32 10-9 19-20 30-29 5-5 11-8 17-11 14-6 26-3 37 5 12 9 23 21 33 32 13 12 25 25 40 34 20 12 41 16 63 7 12-4 21-11 31-19 14-12 26-25 39-38 7-7 14-14 23-19 12-7 24-7 37 0l13 9Zm-357 40 1 1c11 15 24 30 40 41 12 9 25 15 40 17 28 4 51-6 73-25-12-6-22-14-32-22l-3 2c-18 11-35 13-54 0-11-7-20-17-29-27l-10-14-26 27Z"/></svg>
```

Comment (sn00py1310) [reply img/d/desmos.com.svg]: @Carlgo11 thanks for the tip

Comment (phallobst) [reply img/d/desmos.com.svg]: > I think you got that logo from `desmos.com.au`...

I just wanted to point out that the logo was upside-down, but the joke fell flat, I guess. :man_shrugging:

Comment (sn00py1310) [reply img/d/desmos.com.svg]: Oh, and I tried everything to reach it but the domain wasn't sold.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**